content/handbook/people-group/orientation-identity.md

```
@@ -0,0 +1,229 @@
+---
+title: "ジェンダーと性的指向アイデンティティの定義と FAQ"
+description: "ジェンダーと性的アイデンティティの定義、よくある質問、リソース"
+upstream_path: /handbook/people-group/orientation-identity/
+upstream_sha: 1e195b58b9f249ff10bd0e705106c320fee86141
+translated_at: "2026-05-14T00:00:00Z"
+translator: claude
+stale: false
+---
+
+このページのコンテンツは、用語集と FAQ を持つ[他のサイト](#additional-resources)に着想を得ています。多くの LGBTQ+ 用語には完全な統一はなく、コミュニティによって異なる意味やニュアンスを持つ場合があることに留意してください。これは広く受け入れられている定義を持つ用語集です。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)